packages/-ember-data/tests/integration/identifiers/configuration-test.ts

```diff
@@ -22,7 +22,8 @@ import Store, {
 import { identifierCacheFor } from '@ember-data/store/-private';
 
 type StableRecordIdentifier = import('@ember-data/store/-private/ts-interfaces/identifier').StableRecordIdentifier;
-type ExistingResourceObject = import('@ember-data/store/-private/ts-interfaces/ember-data-json-api').ExistingResourceObject;
+type ExistingResourceObject =
+  import('@ember-data/store/-private/ts-interfaces/ember-data-json-api').ExistingResourceObject;
 
```

Comment:
in-range updates to lint and prettier enforce the max-line-length for these type declarations now (they didn't before) which is why some of them now split lines.